test/dialogmoteunntak/DialogmoteunntakSkjemaTest.tsx

```diff
@@ -63,56 +57,61 @@ describe("DialogmoteunntakSkjema", () => {
       .empty;
   });
 
-  it("valideringsmeldinger forsvinner ved utbedring", () => {
+  it("valideringsmeldinger forsvinner ved utbedring", async () => {
     renderDialogmoteunntakSkjema();
     clickButton(submitButtonText);
 
-    const arsakErrorText = "Vennligst angi årsak";
-    expect(screen.getAllByText(arsakErrorText)).to.not.be.empty;
+    await waitFor(() => {
```

Comment:
Her kunne man kanskje forenklet til
`expect(await screen.findByText(unntakSkjemaTexts.arsakErrorMessage)).to.not.be.empty;`

Reply:
Pent!